part-time-schedule/week-9.md

```diff
@@ -0,0 +1,28 @@
+# Week 8
+
+## Primary focus
+
+This week the primary focus will be on Open Source Projects.
+
+### Monday
+
+We will start the week by having a Contributing Etiquette Workshop that will prepare us on how to 
+contribute to Open Source Projects. It will also look at polishing our Projects on Github 
+covering Readme, Licence, Contribution guidelines amongst others.
```

Comment:
I think we expected to have them done with the GitHub and project polishing with the extra time they received last week.

Reply:
yes, most of them did. Not sure it's all of them. I will take it out then.